src/main/java/com/exasol/projectkeeper/validators/files/ProjectFilesValidator.java

```diff
@@ -74,13 +76,14 @@ private void fixFile(final File projectFile, final Resource templateFile) {
         }
     }
 
-    private void validateContent(final FileTemplate template, final Consumer<ValidationFinding> findingConsumer,
-            final File projectFile) {
+    private List<ValidationFinding> validateContent(final FileTemplate template, final File projectFile) {
         if (!isFileEqualWithTemplate(template, projectFile)) {
-            findingConsumer.accept(new ValidationFinding(
+            return List.of(new ValidationFinding(
                     ExaError.messageBuilder("E-PK-18").message("Outdated content: {{file name}}")
                             .parameter("file name", template.fileName).toString(),
-                    (Log log) -> fixFile(projectFile, template.template)));
+                    (Log log) -> fixFile(projectFile, template.template.getPath())));
+        } else {
+            return List.of();
```

Comment:
You returned Collections.emptyList() in some other places. Maybe it's good to unify them?

Reply:
:+1: 
I replaced it everywhere now